src/mpid/common/sched/mpidu_sched.c

```diff
@@ -136,13 +136,19 @@ struct MPIDU_Sched_state {
 };
 
 /* holds on to all incomplete schedules on which progress should be made */
-struct MPIDU_Sched_state all_schedules = { NULL };
+static struct MPIDU_Sched_state all_schedules = { NULL };
```

Comment:
I think we should add a namespace to `all_schedules` just like we did for `coll_queue`.

Reply:
We don't need namespace for static variables.

Reply:
Got it